.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,6 +14,6 @@
 - [ ] The page has 8 or fewer examples.
 
 - [ ] The PR is appropriately titled:  
-      "command: add page" for new pages, or "command: description of changes" for pages being edited
+      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited
```

Comment:
Perhaps we should change the `<` `>` out for `{{` and `}}`, as we have in the pages themselves?

Reply:
I'd rather not. Angle brackets are quite common as a markup for placeholders, while the double curly brackets are quite specific to tldr-pages, and were chosen because of technical difficulties (to avoid conflicts with redirection operators), rather than because they were considered better than the angle brackets. But open to hear other arguments.

Reply:
I am neutral about this. Both are fine by me.

Reply:
Ah, ok then :smiley_cat: